### PR TITLE
Update in tool that searches for usage of deprecated methods.

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

**Comment by awholegunch** [comment]
"On-hold" because a change has been requested for this tool.

File: cdap-docs/tools/docs-search-deprecated.py [hunk 1]
```diff
@@ -42,91 +43,150 @@ def parse_options():
...
+                longest = len(deprecated)
+    return deprecated_items, longest
+
+def _load_deprecated_items(deprecated_url):
+
     print "Loading deprecated info from '%s'" % deprecated_url
     page = urllib2.urlopen(deprecated_url).read()
     soup = BeautifulSoup(page, 'html.parser')
-    soup.prettify()
+#     soup.prettify()
```
Comment (anwar6953) [inline cdap-docs/tools/docs-search-deprecated.py]: can we just remove it?

Comment (awholegunch) [reply cdap-docs/tools/docs-search-deprecated.py]: Let me change it to a better comment; I don't want to forget this...

Comment (awholegunch) [reply cdap-docs/tools/docs-search-deprecated.py]: Actually changed my mind and removed it as you suggested; it wasn't what I was thinking of.

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**